ts/src/bitmex.ts

```diff
@@ -938,17 +1087,17 @@ export default class bitmex extends Exchange {
         }
         let feeCost = this.safeNumber (item, 'fee', 0);
         if (feeCost !== undefined) {
-            feeCost = feeCost / 100000000;
+            feeCost = this.convertToRealAmount (code, feeCost);
         }
         const fee = {
             'cost': feeCost,
             'currency': code,
         };
         let after = this.safeNumber (item, 'walletBalance');
         if (after !== undefined) {
-            after = after / 100000000;
+            after = this.convertToRealAmount (code, after);
         }
-        const before = this.sum (after, -amount);
+        const before = this.parseNumber (Precise.stringAdd (this.numberToString (after), Precise.stringNeg (this.numberToString (amount))));
```

Comment:
```suggestion
        const before = this.parseNumber (Precise.stringSub (this.numberToString (after), this.numberToString (amount)));
```
maybe shorter?

Reply:
@ttodua yeah yeah I was trying to keep the same logic but with string math but your version is shorter